src/schema/payment.ts

```diff
@@ -139,27 +127,31 @@ export interface RecurringPaymentSearchParams extends SearchParams {
 }
 
 type RecurringPaymentStatus =
```

Comment:
So i think we only return these statuses: https://openbankinguk.github.io/read-write-api-site3/v3.1.11/resources-and-data-models/vrp/domestic-vrps.html#status

The longer list is if we get the `paymentDetails` and i am not too sure if we add that to the normal status. I can have a look and return to this comment

Reply:
Have used the condensed list for now